azure-pipelines.yml

```diff
@@ -34,14 +34,29 @@ pr:
 stages:
            
   - stage: system_tests_win64
+    dependsOn: []
     pool:
       name: nimi-bot
       demands:
         - nimi-bot -equals True
-        - agent.os -equals Windows_NT
+        - platform -equals win64 # Run system tests using python 64 bit installed on windows 64 bit
     jobs:
     - ${{each module_name in parameters.module_names}}:
       - template: run_module_system_tests.yml
         parameters:
           module_name: ${{module_name}}
           platform: win64
+
+  - stage: system_tests_win32
+    dependsOn: []
+    pool:
+      name: nimi-bot
+      demands:
+        - nimi-bot -equals True
+        - platform -equals win32 # Run system tests using python 32 bit installed on windows 64 bit
+    jobs:
+    - ${{each module_name in parameters.module_names}}:
+      - template: run_module_system_tests.yml
+        parameters:
+          module_name: ${{module_name}}
+          platform: win64
```

Comment:
was this supposed to say `win32`

Reply:
also note you probably should have a newline afterwards.

Reply:
I have removed platform from the run_module_system_tests.yml file